plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""The entropy hasher implementation."""
+
+from __future__ import unicode_literals
+
+import codecs
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
+  """This class provides entropy hashing functionality."""
```

Comment:
Entropy of what? byte value distribution?

Reply:
Done